modules/op-customizing-pipelines-as-code-configuration.adoc

```diff
@@ -19,12 +19,30 @@ To customize {pac}, cluster administrators can configure the following parameter
 
 | `max-keep-days` | The number of the days for which the executed pipeline runs are kept in the `pipelines-as-code namespace`. 
 
-Note that this configmap setting does not affect the cleanups of a user's pipeline runs, which are controlled by the annotations on the pipeline run definition in the user's GitHub repository. |
+Note that this configmap setting does not affect the cleanups of a user's pipeline runs, which are controlled by the annotations on the pipeline run definition in the user's GitHub repository. |  NA
 
 | `secret-auto-create` | Indicates whether or not a secret should be automatically created using the token generated in the GitHub application. This secret can then be used with private repositories. | `enabled` 
 
 | `remote-tasks` | When enabled, allows remote tasks from pipeline run annotations. | `enabled`
```

Comment:
IBM SG recommends against using "allows" for inanimate objects, but I think it is fine here - otherwise you'd just end up using a synonym anyways.